faculty/clients/job.py

```diff
@@ -256,6 +348,21 @@ def list(self, project_id):
         endpoint = "/project/{}/job".format(project_id)
         return self._get(endpoint, JobSummarySchema(many=True))
 
+    def get_job(self, project_id, job_id):
```

Comment:
```suggestion
    def get(self, project_id, job_id):
```
To match `list` and the semantics of methods on other clients (methods for runs and subruns _do_ incude the name to distinguish them).